app/build.gradle

```diff
@@ -175,6 +172,7 @@ android {
     }
 
     packagingOptions {
+        resources.excludes.add("META-INF/**/*")
```

Comment:
I think that we don't need these wildcards, with these excludes should work:
`exclude 'META-INF/proguard/androidx-annotations.pro'
        exclude 'META-INF/gradle/incremental.annotation.processors'`